### PR TITLE
Bog ranger choice

### PR DESCRIPTION
**Pull request by Rayforx**
## About The Pull Request
It makes the bog ranger a choice, instead of a randomly selected
It still has the same limit of only being >3 players
The first to choose the 3 rangers gets the role
Change the name of the bog crossbowman to ranger, since it fits and females can get it

+ Removed females being forced into certain hairstyles, a bit weird to have your hair changed as a vulp
+ Added a cloak to ranger for STYLE while they go alone in the wilderness
+ Removed a bug where the crossbowman would get LEGENDARY SKILLS since they got the skill points from both the bog guard and the crossbowman skills

## Why It's Good For The Game
Not having to play a ranger, if you do not want to for the shift
Having the choice to be something, so your character can be known as a skilled bowman

### CODE REVIEW

**Comment by RadiantFlash7** [comment]
Oh, Yes please. Theres nothing worse then joining a role and having to HOPE for the 35% chance of getting what you want, if you want your character to be an archer.

**Comment by wow50000** [comment]
could you do the same with town guards too ?

**Comment by Rayforx** [comment]
> could you do the same with town guards too ?

It would have to be in another PR, but yes

File: code/modules/jobs/job_types/roguetown/garrison/bogguard.dm [hunk 1]
```diff
@@ -65,22 +80,24 @@
...
+
+/datum/outfit/job/roguetown/ranger/pre_equip(mob/living/carbon/human/H)
+	. = ..()
+	backr = /obj/item/storage/backpack/rogue/satchel
+	backl = /obj/item/gun/ballistic/revolver/grenadelauncher/bow
+	beltr = /obj/item/quiver/arrows //replaces sword
+	wrists = /obj/item/clothing/wrists/roguetown/bracers/leather
+	backpack_contents = list(/obj/item/rogueweapon/huntingknife/idagger/steel = 1, /obj/item/clothing/cloak/raincloak/brown = 1 )
+	ADD_TRAIT(H, TRAIT_MEDIUMARMOR, TRAIT_GENERIC)
+
```
Comment (RadiantFlash7) [inline code/modules/jobs/job_types/roguetown/garrison/bogguard.dm:92]: Missing 
if(H.min)
assign_skills(h)

**Comment by RadiantFlash7** [comment]
Currently test merged on the server, ranger is missing the two lines that call the skill proc. Currently testing it, and no skills at all.

**Comment by Rayforx** [comment]
The bug was fixed

**Comment by HissyNays** [comment]
Yes please, more of this. Town guard needs it as stated above it. Good work.

**Comment by CalamaBanana** [comment]
Please do capitalize the name of the subclasses, noticed that in the steward's console. Looked a bit ugly with them being the only ones in lower case.

(Also, just personal preference, maybe not "Ranger"? The word Ranger invites people to think they should be ranging in the bog instead of guarding the gate. But I don't have any better idea myself.)

**Comment by Rayforx** [comment]
> Please do capitalize the name of the subclasses, noticed that in the steward's console. Looked a bit ugly with them being the only ones in lower case.
> 
> (Also, just personal preference, maybe not "Ranger"? The word Ranger invites people to think they should be ranging in the bog instead of guarding the gate. But I don't have any better idea myself.)

I will see about capitalizing them

But in terms of name, then I feel that it fits well with calling them ranger since the bog master still has to scout the bog/ check for tunnels/ and perform other scout duties, which makes sense to have a *ranger* do it

**Comment by Rayforx** [comment]
image
Picture of changes

**Comment by CalamaBanana** [comment]
> Picture of changes

Thanks! Looks much cleaner.

**Pull request merged**